docs/scenarios_authoring/Fragments.md

```diff
@@ -9,14 +9,34 @@ Fragments are a way to share processing logic - if many scenarios require identi
 Once a fragment is created using Designer (see Fragments tab) it can be used in other scenarios in the same category.
 
 ## Inputs
-Fragment can have one input. You can define parameters of a fragment:
+Fragment must have exactly one input. You can define parameters of a fragment:
 
-![fragment input](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_fragments_-_Inputs0.png)
+![fragment input_base](img/fragment_input_simple.png)
+
+Clicking the "Show advanced configuration" button opens up an additional form, that allows changing the parameter's behaviour during inputting its value when using the fragment, e.g. changing its validation requirements or displaying a hint.
```

Comment:
If we use term options (parameter options) rather than original phrasing, I would change the screen hint associated with options button. It is `SettingsButton` (without space) currently and is more appropriate for UI design tool. Should be `Settings` or `Options`.